Tests/SwiftParserTest/Assertions.swift

```diff
@@ -275,15 +275,13 @@ class FixItApplier: SyntaxRewriter {
   var changes: [FixIt.Change]
 
   init(diagnostics: [Diagnostic], withMessages messages: [String]?) {
+      let messages = messages ?? diagnostics.compactMap { $0.fixIts.first?.message.message }
```

Comment:
```suggestion
    let messages = messages ?? diagnostics.compactMap { $0.fixIts.first?.message.message }
```

Again: Didn’t swift-format catch this?